src/test/java/org/openrewrite/java/micronaut/UpdateMavenAnnotationProcessorsTest.java

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.micronaut;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.config.Environment;
+import org.openrewrite.test.RecipeSpec;
+
+import static org.openrewrite.java.Assertions.mavenProject;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+public class UpdateMavenAnnotationProcessorsTest extends Micronaut4RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.recipes(Environment.builder()
+          .scanRuntimeClasspath("org.openrewrite.java.micronaut")
+          .build()
+          .activateRecipes(
+            "org.openrewrite.java.micronaut.UpdateMicronautPlatformBom",
+            "org.openrewrite.java.micronaut.UpdateBuildPlugins",
+            "org.openrewrite.java.micronaut.UpdateMavenAnnotationProcessors"));
+    }
+
+    @Test
+    void updateCoreMavenAnnotationProcessors() {
+        rewriteRun(mavenProject("project",
```

Comment:
maybe a nitpick, but it's likely not necessary to add `mavenProject("project",` before your pom.xml; might save you a bit of setup per test, and then it stands out more when it _is_ necessary.

Reply:
Ahh ok cool...not sure why I got in the habit of always including it.